src/js/themes/base.d.ts

```diff
@@ -278,6 +278,9 @@ export interface ThemeType {
         duration?: string;
       };
     };
+    backgrounds?: {
+      [x: string]: BackgroundType | { dark?: string; light?: string };
```

Comment:
```suggestion
      [x: string]: BackgroundType | { dark?: string; light?: string } | string;
```

Reply:
Since BackgroundType includes string in its definition, do we need the additional string?

```
export type BackgroundType = string | BackgroundObject;
```

Reply:
Thank you, missed this detail.